docs(vpc): requirements update (#2945)
scaleway/docs-content: network/vpc/how-to/activate-dhcp.mdx

Check out our [migration](/network/vpc/reference-content/vpc-migration/) documentation for full information about how "old" Private Networks were migrated when VPC went into General Availability, and for more information about DHCP.
</Message>

> We highly recommend activating DHCP on all of your Private Networks, to let us take care of IP address management in our fully-integrated ecosystem. This document, therefore, explains how to activate DHCP on Private Networks where it has not been automatically activated.

Reviewer: ```suggestion
We highly recommend activating DHCP on all of your Private Networks, to let us take care of IP address management in our fully-integrated ecosystem. This page explains how to activate DHCP on Private Networks where it has not been automatically activated.
```

</Message>

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization

> - A Private Network created prior to July 12 2023, in which DHCP is not yet activated

Reviewer: ```suggestion
- Created a Private Network in which DHCP is not yet activated, before July 12, 2023
```
